packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx

```diff
@@ -302,6 +306,17 @@ const StakeExchangeForm = ({
     ? false
     : numberOfStakeToExchange >= stakeBalance;
 
+  const handleClickCopyClipboard = (address: string) => {
+    navigator.clipboard
```

Comment:
please use `saveToClipboard` util we have in the codebase. It also supports notification toast